source/_integrations/modbus.markdown

```diff
@@ -48,6 +48,11 @@ modbus:
 ```
 
 {% configuration %}
+close_comm_on_error:
+  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters sends garble to HA when opened, this leads to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```

Comment:
```suggestion
  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters send garble to HA when opened, this leads to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```

Reply:
I will not let me accept your suggestion, but please commit the change.

Reply:
I already committed it, hence you couldn't accept it 😉